lib/sinon/proxy.js

```diff
@@ -40,7 +40,11 @@ var proxyApi = {
         return emptyFakes;
     },
 
-    getCall: function getCall(i) {
+    getCall: function getCall(index) {
+        var i = index;
+        if (i < 0) {
+            i += this.callCount;
```

Comment:
Delightfully simple. Wonderful!

```suggestion
            // Negative indices means counting backwards from the last call
            i += this.callCount;
```

Reply:
Applied the comment (I'm quite new to GitHub, assume that's the right way to do it, I have the impression there's some automagic to apply comments with a click, but only with the right permissions)